standard/expressions.md

```diff
@@ -549,7 +549,7 @@ argument_name
 argument_value
     : expression
     | 'ref' variable_reference
-    | 'out' variable_reference
+    | 'out' declaration_expression? identifier
```

Comment:
While the idea of factoring out to a common shared production is not wrong *per se* in this case doing so has changed the grammar ☹️. Revert to the intermediate version:

```suggestion
    | 'out' local_variable_type? identifier
```